kyuubi-ctl/src/test/scala/org/apache/kyuubi/ctl/KyuubiCtlArgumentsSuite.scala

```diff
@@ -201,4 +201,35 @@ class KyuubiCtlArgumentsSuite extends KyuubiFunSuite {
     assert(opArgs3.verbose)
     assert(opArgs3.action == KyuubiCtlAction.HELP)
   }
+
+  test("test with unknown host") {
+    val args = Array(
+      "create", "server",
+      "-zk", zkAddress,
+      "-ns", namespace,
```

Comment:
I'm not sure if it's a good arg style. personally, I prefer
- single-letter Unix-style, i.e. `-v -p 1234` or `-v -p1234` or `-vp1234`
- all-word  GNU-style, i.e. `--verbose --port 1234` or `--verbose --port=1234`

But `java -version` dose not use that style

Reply:
```
Arguments:
  --zkAddress, -zk            one of the zk ensemble address, using kyuubi-defaults/conf
                              if absent
  --namespace, -ns            the namespace, using kyuubi-defaults/conf if absent
  --host, -h                  hostname or IP address of a service
  --port, -p                  listening port of a service
  --version, -V               using the compiled KYUUBI_VERSION default, change it if the
                              active service is running in another
  --user, -u                  for engine service only, the user name this engine belong to
  --help, -I                  Show this help message and exit.
  --verbose, -v               Print additional debug output.
```

Reply:
Yes, I saw that, but we haven't release v1.2.0, if we have a better choice, the change is safe.

Reply:
do you mean that, align `--version` with `-v`?

Reply:
I suggest 

- change `--zkAddress, -zk` to `--zk-addr, -z`
- change `--help, -I` to `--help, -I`

to make it match more command line tool arg styles.

also cc @yaooqinn

Reply:
could you clear it for `--help`? @pan3793

Reply:
Well, I am not a fan of abbreviations, while it always confusing me such as `-h` here is for help or short. Shall we only use fully qualified parameters for v1.2.0?

Reply:
@turboFei sorry for typo, fixed

Reply:
sure, we can use full qualified parameters at first.

Reply:
@yaooqinn I'm OK with only GNU-style, especially in shell script, the single-letter arg is not clear sometimes.

Reply:
should we change `--zkAddress` to `--zk-addr`?
cc @yaooqinn

Reply:
on second thought, it's better to use `--zk-quorum` to match with `kyuubi.ha.zookeeper.quorum`

Reply:
+1 for `--zk-quorum`, please also change the text, `one of the zk ensemble address` is not properly here